lib/world_cup.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/team'
+require './lib/player'
+
+class WorldCup
+
+  attr_reader :year,
+              :teams
+
+  def initialize(year, teams)
+    @year = year
+    @teams = []
+    teams.each do |team|
+      @teams << team
+    end
+  end
+
+  def active_players_by_position(search_position)
+    active_players = []
+    @teams.each do |team|
+      if !team.eliminated?
+        active_players = team.players_by_position(search_position)
+      end
+    end
+    active_players
```

Comment:
Think that this only returns the players from the last team that is in `@teams`. In order to get all of the active players for a position would need to shovel the result of `team.players_by_position(search_position)` into `active_players` and then flatten the `active_players` array before returning it.